assets/js/theme/category.js

```diff
@@ -18,6 +18,19 @@ export default class Category extends CatalogPage {
             });
         });
 
+        if ($('[data-shop-by-price]').length) {
+            if ($('.navList-action').hasClass('is-active')) {
+                $('a.navList-action.is-active').focus();
+            }
+
+            $('a.navList-action').on('click', () => {
```

Comment:
As I can see it will be applied for all sidebar filters, do we need this only when shop by price is available?

Reply:
yes, it's needed for shop by price filter only so I put it under if condition (line23 after last change). and this event handler won't be applied for other sidebar fitlers.